Hackney.Shared.Finance.Tests/Dockerfile

```diff
@@ -0,0 +1,24 @@
+﻿FROM mcr.microsoft.com/dotnet/core/sdk:3.1
```

Comment:
The base docker image does not match our projects dotnet version.
Interestingly enough, this doesn't seem to be an issue for the build process. :/